extensions/wikia/AdEngine/js/slot/bottomLeaderboard.js

```diff
@@ -18,6 +18,7 @@ define('ext.wikia.adEngine.slot.bottomLeaderboard', [
 		logGroup = 'ext.wikia.adEngine.slot.bottomLeaderboard',
 		pushed = false,
 		wikiaFooter,
+		btRec = babDetection.isBlocking() && wadRecRunner.isEnabled('bt'),
```

Comment:
Bottom_Leaderboard is injected only once, so we don't need to move it to separate variable.